docs/caching/includes/azure-redis-app-host.md

```diff
@@ -21,20 +19,147 @@ dotnet add package Aspire.Hosting.Azure.Redis
+  - A tag is added to the Redis resource with a key of `ExampleKey` and a value of `Example value`.
+
+There are many more configuration options available to customize the Azure Cache for Redis resource. For more information, see <xref:Azure.Provisioning.Redis>. For more information, see [Azure.Provisioning customization](../../azure/integrations-overview.md#azureprovisioning-customization).
+
+### Connect to an existing Azure Cache for Redis
+
+You might have an existing Azure Cache for Redis that you want to connect to. Instead of representing a new Azure Cache for Redis resource, you can add a connection string to the app host. To add a connection to an existing Azure Cache for Redis, call the <xref:Aspire.Hosting.ParameterResourceBuilderExtensions.AddConnectionString*> method:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var cache = builder.AddConnectionString("azure-redis");
+
+builder.AddProject<Projects.WebApplication>("web")
+       .WithReference(cache);
+
+// After adding all resources, run the app...
+```
+
+[!INCLUDE [connection-strings-alert](../../includes/connection-strings-alert.md)]
+
+The connection string is configured in the app host's configuration, typically under [User Secrets](/aspnet/core/security/app-secrets), under the `ConnectionStrings` section. The app host injects this connection string as an environment variable into all dependent resources, for example:
+
+```json
+{
+    "ConnectionStrings": {
+        "azure-redis": "localhost:6379,ssl=true,abortConnect=False"
+    }
+}
+```
+
+The dependent resource can access the injected connection string by calling the <xref:Microsoft.Extensions.Configuration.ConfigurationExtensions.GetConnectionString*> method, and passing the connection name as the parameter, in this case `"azure-redis"`. The `GetConnectionString` API is shorthand for `IConfiguration.GetSection("ConnectionStrings")[name]`.
+
+### Run Azure Cache for Redis resource as a container
+
+The Azure Cache for Redis hosting integration supports running the Redis server as a local container. This is beneficial for situations where you want to run the Redis server locally for development and testing purposes, avoiding the need to provision an Azure resource or connect to an existing Azure Cache for Redis server.
+
+To make use of the [`docker.io/library/redis`](https://hub.docker.com/_/redis/) container image, and run the Azure Cache for Redis instance as a container locally, chain a call to <xref:Aspire.Hosting.AzureRedisExtensions.RunAsContainer*>, as shown in the following example:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var cache = builder.AddAzureRedis("azcache")
+                   .RunAsContainer();
+
+builder.AddProject<Projects.ExampleProject>()
+       .WithReference(cache);
+
+// After adding all resources, run the app...
+```
+
+The preceding code configures an Azure Cache for Redis resource to run locally in a container.
```

Comment:
This isn't true, and it is why we call it `RunAsContainer` and not `RunAsEmulator`. This isn't the "Azure Cache for Redis" running locally in a container. It is just a normal Redis container.

```suggestion
The preceding code configures the Redis resource to run locally in a container.
```